### PR TITLE
About us 

### LINKED ISSUE #18: Update AboutUs page

### PR DESCRIPTION
**Pull request by marcusjhang**

### CODE REVIEW

File: docs/team/marcusjhang.md
Comment (notnotmax) [inline docs/team/marcusjhang.md:1]: Maybe remove the placeholder text? You can add to it as we work on the tP.

File: docs/AboutUs.md
Comment (notnotmax) [inline docs/AboutUs.md:1]: Maybe you can pick a responsibility you'd like to be in charge of and update it accordingly?

**Review by notnotmax — COMMENTED** [review]
lgtm besides minor issues

**Review by S-K-Y-Light — APPROVED** [review]
LGTM

**Pull request merged**